routes/tasks.js

```diff
@@ -40,4 +40,6 @@ router.patch(
 );
 router.patch("/assign/self", authenticate, invalidateCache({ invalidationKeys: [ALL_TASKS] }), tasks.assignTask);
 
+router.post("/migration", authenticate, authorizeRoles([SUPERUSER]), tasks.updateStatus);
+
```

Comment:
This endpoint is not clear to me look for better conventions

Reply:
Thank you for your feedback regarding the /migration endpoint. The term "migration" has been intentionally selected to align with the semantic meaning of a one-time data transformation or update.
If you have any further concerns or suggestions for improvement, I am open to discussion.